tmp/metrics/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Isn't `tmp/*` just ignored?

Reply:
The problem is that if `/tmp/metrics` does not exist, then a lot of errors are raised in the log. This is to make sure that the directory exists after repo is cloned and user does not need to create it manually.

Reply:
You can just use `.gitkeep`, but wondering, can't you create the directory before running the metrics. it is only for IntelliJ right? Also if so they should fix that in their plugin.